app/views/wallet/TransactionDetailsView/TransactionDetailsView.tsx

```diff
@@ -64,11 +64,17 @@ const TransactionDetailsView: FC<TransactionDetailsViewProps> = ({
   const sign = direction === 'tx_direction_sent' ? '-' : '+';
 
   // TODO replace this with a component
-  const renderRow = (title: string, value: string | ReactNode) => (
+  const renderRow = (title: string, value: string | ReactNode, txo?: boolean) => (
```

Comment:
If this `txo` is a boolean, it should be `hasTxo`. But I'm not seeing where `renderRow` is being changed to include this `txo` anyway?